From 6ee557360645da158e3065111349c318894b497b Mon Sep 17 00:00:00 2001
From: Steve Markgraf <steve@steve-m.de>
Date: Mon, 10 Feb 2014 23:59:21 +0100
Subject: [PATCH 5/5] rtl_tcp: make all global variables static

Signed-off-by: Steve Markgraf <steve@steve-m.de>
---
 src/rtl_tcp.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/rtl_tcp.c b/src/rtl_tcp.c
index a54666a..317e0f3 100644
--- a/src/rtl_tcp.c
+++ b/src/rtl_tcp.c
@@ -78,9 +78,9 @@ typedef struct { /* structure size must be multiple of 2 bytes */
 
 static rtlsdr_dev_t *dev = NULL;
 
-int global_numq = 0;
+static int global_numq = 0;
 static struct llist *ll_buffers = 0;
-int llbuf_num=500;
+static int llbuf_num = 500;
 
 static volatile int do_exit = 0;
 
-- 
1.7.10.4

